components/notification/NotificationEnablePushNotification.client.vue

```diff
@@ -66,7 +66,10 @@ const isLegacyAccount = computed(() => !currentUser.value?.vapidKey)
       :disabled="busy || isLegacyAccount"
       @click="$emit('subscribe')"
     >
-      <span aria-hidden="true" :class="busy && animate ? 'i-ri:loader-2-fill animate-spin' : 'i-ri:check-line'" />
+      <div v-if="busy && animate" aria-hidden="true" animate-spin preserve-3d>
+        <div i-ri:loader-2-fill aria-hidden="true" />
+      </div>
+      <div v-else aria-hidden="true" i-ri:check-line />
```

Comment:
would you update here and elsewhere so we're not change the elements from span -> div in this PR?

Reply:
@danielroe @sxzz I'll keep original elements updating only span that requires block: the icon not being displayed when using span inside span.

Reply:
can we find another solution for that? we shouldn't have any `<div>` inside `<button>` - it's not permitted HTML content

Reply:
yeah, I get this warning using IntelliJ, all my components using span inside button

Reply:
I'll revert last commit, you need to review a few components...

Reply:
done